content-filter/tests/case-01.http

```diff
@@ -0,0 +1,20 @@
+POST http://localhost:8080/payroll/employees/907f059924cc1acad24e950ef7a9b249/paytemplate/reimbursements
+
+[
+    {
+        "reimbursementTypeID": "83791a58-e043-4883-ad53-6f4cba62c313",
+        "reimbursementTypeName": "GST",
```

Comment:
Use meaningful type names and names. eg type names "Travel Allowance" "Parking Expenses".